forms-flow-web/src/components/ServiceFlow/list/sort/TaskAttributeComponent.js

```diff
@@ -189,14 +189,15 @@ function TaskAttributeComponent({
                   {(inputValues.length - 1 === index) ? (
                     <button
                       type="button"
+                      disabled={!inputValues[index].name.length || !inputValues[index].label.length}
                       className="btn btn-primary mt-3"
                       onClick={() => handleAddClick()}
```

Comment:
don't need to write an arrow function inside the **onclick** because it will create extra execution context in call stack . if no arguments expected in **handleAddClick** function we just give the function inside the onClick ` onClick={handleAddClick}`